apps/emloyees-management/src/app/home/home.component.html

```diff
@@ -0,0 +1,3 @@
+<p>home component works!</p>
```

Comment:
please remove default text

Reply:
It's not so important for the current iteration. I'll do it through the next steps.